Code/GraphMol/Atom.cpp

```diff
@@ -517,6 +517,80 @@ void Atom::invertChirality() {
       break;
   }
 }
+
+namespace MDL {
```

Comment:
Sounds good.  I was playing with better exporting of namespaces in C++ and python, but probably went a bit overboard.

What I like about the namespaces is that these functions are tied into a a particular file format and convey that weight, i.e. using something like:

```
MolFileProps::setAtomAlias
```

has a bit more meaning to me.  Otherwise the details need to be in the doc strings which may be somewhat hidden.